### ISSUE TITLE
Model into module : cannot load such file -- /jets_apps/test/app/models/my_module (LoadError)

### ISSUE THREAD

**Issue by maxime-burriez** [tongueroo/jets#267]
Hi,

/!\ No problem with Jets 1.8.14 !

## My Environment

| Software         | Version |
| ---------------- | ------- |
| Operating System | macOS 10.14.5 |
| Jets             | 1.9.5 |
| Ruby             | 2.5.5 |

---

## Expected Behaviour

Load `app/models/my_module/my_model.rb`.

## Current Behavior

`my_module` folder is considered as a loading file.

## Step-by-step reproduction instructions

To reproduce, follow this steps:

1) Create a new project : `jets new test` (*with Jets 1.9.5*)

2) Add this model : `app/models/my_module/my_model.rb`

```
# frozen_string_literal: true

module MyModule
  class MyModel < ApplicationRecord
  end
end
```

3) Run `jets c`

4) We have this error:

> `require': cannot load such file -- /jets_apps/test/app/models/my_module (LoadError)

Thank you very much !

**Comment by tongueroo** [comment]
Thanks for the bug report. Fixed in #269 Details in there. Released in [v1.9.6](https://github.com/tongueroo/jets/blob/master/CHANGELOG.md)

**Comment by maxime-burriez** [comment]
Thank you very much for your responsiveness and your fix! :ok_hand:
